README.md

```diff
@@ -164,6 +165,19 @@ Any customer projects accessible by end users need to support whatever browsers
 
 When adding support for additional browsers infers very little overhead, we should support as broadly as possible. The two latest versions of all major browsers is the industry standard. No version of Internet Explorer is formally supported.
 
+#### System identification
+
+All outgoing HTTP requests should identify the caller via a properly formatted `User-Agent` header. The first identifier in the header should be `aptoma`, followed by service name and version. Additionally, services may add arbitrary data as key-value pairs in parentheses, of which the minimum requirement is to describe the deployment environment. This format is specified by [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#field.user-agent).
```

Comment:
```suggestion
All outgoing HTTP requests should identify the caller via a properly formatted `User-Agent` header. The first identifier in the header should be `aptoma`, followed by service name and optional version. Additionally, services may add arbitrary data as key-value pairs in parentheses, of which the minimum requirement is to describe the deployment environment. This format is specified by [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#field.user-agent).
```


I think it's generally a good idea to always include a version, and that it should always be possible to pull a version from either a version number or commit SHA, but if there is some case where this impractical, I don't think we need to be dogmatic about it. The most important is that we and others are able to identify who made a request.

Reply:
^ Based on Zulip discussions, by the way.